returnn/oggzip.py

```diff
@@ -57,9 +59,13 @@ def __init__(
             returnn_root if returnn_root is not None else gs.RETURNN_ROOT
         )
 
+        self.rqmt = rqmt
```

Comment:
The default style is to put self.rqmt after the output definitions at the end of the constructor.